test/unit/TestRunnerOrchestratorSpec.ts

```diff
@@ -17,7 +17,12 @@ let expect = chai.expect;
 describe('TestRunnerOrchestrator', () => {
   let sut: TestRunnerOrchestrator;
   let sandbox: sinon.SinonSandbox;
-  let files = [{ path: 'a.js', shouldMutate: true }, { path: 'b.js', shouldMutate: true }, { path: 'aSpec.js', shouldMutate: false }, { path: 'bSpec.js', shouldMutate: false }];
+  let files = [
+    { path: process.cwd() + path.sep + 'a.js', shouldMutate: true },
+    { path: process.cwd() + path.sep + 'b.js', shouldMutate: true },
+    { path: process.cwd() + path.sep + 'aSpec.js', shouldMutate: false },
+    { path: process.cwd() + path.sep + 'bSpec.js', shouldMutate: false }
```

Comment:
Ignore this comment. As it is a unit test testing an other unit, it is not necessary